netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
No we haven't addedd the API version yet.

Btw, I wasn't suggesting that showing version would ensure compatibility, rather a way to compare if things are compatible. I think it will come down to issuing two commands and comparing them, vs netctl version.

If folks feel strongly that 'netctl version' is an overkill, I can resubmit the PR removing the code for `netctl version`. If we don't need `netctl version` I can also simplify the Get and Print functions in version.go as one i.e. do not expose Print at all, Get will can give a formatted string. Please let me know.